src/modules/domains/domains.service.ts

```diff
@@ -473,18 +473,18 @@ export class DomainsService {
       );
     }
 
-    const steps = changeOwnerNamespaces.map((namespace) => {
```

Comment:
@jrhender This is a fix for an error involving a map. I personally think its better to call a spade a spade and say what it is, rather than come up with different names for the argument, just to avoid repeating, but that's just my thought. What do you think? cc @JGiter

Reply:
You can see below a lot of the warnings are for using 'namespace' in a map.

Reply:
With the above change with ignoreFunctionTypeParameterNameValueShadow, we would not have to do this for every .map or every method with a param that has a shared name (like 'namespace')

Reply:
I think lenght of variable name should depend on size of variable scope. This `namespace` is used only for one operation - `changeDomainOwnerTx`, thus I would name it just by `n`, but variable occures several times in order to convert return type of `changeOrgOwnership` to `MulticallTx` (which is also array of `ReturnStepWithRetryCheck`). This conversion happens in other places too. So you can separate conversion from `tx` to `ReturnStepWithRetryCheck` and then the code in question would look like
```
 const steps = changeOwnerNamespaces.map((n) => {
      return this.txToCall(
         this.changeDomainOwnerTx({ newOwner, namespace: n }), 
         ()=>await this.getOwner({namespace: n}) === newOwner
   )
}

txToCall(tx, retryCheck) {...}
```
I would also change `getOwner({namespace})` to `getOwner(namespace)`. This is not breaking change, because `getOwner` is protected

Reply:
I'm also not the biggest fan of single letter variable names, but I think they have their place in loops, closures, in-line functions. I did a bit of SO research and found this post which is saying that single letter variable names are reasonable within a lexical scope of about 25 lines, which seems about right to me. https://stackoverflow.com/a/5803019 
So I think the single letter solution could work here. What do you think @whitneypurdum ?

Reply:
Works for me! What is "SO"?

Reply:
Oh, stackoverflow.